properties/P000146.md

```diff
@@ -4,10 +4,15 @@ name: Ultraparacompact
 refs:
   - doi: 10.48550/arXiv.1306.6086
     name: Ultraparacompactness and Ultranormality (J. Van Name)
-  - mr: 261565
+  - zb: "0182.25501"
     name: Extending continuous functions on zero-dimensional spaces (R. Ellis)
 ---
 
 Every open cover of the space has an open (hence clopen) refinement partitioning the space. (We do not assume {P3}.)
 
-Equivalently, every open cover of the space is refined by a locally finite clopen cover.  The equivalence between the two definitions is shown in Lemma 1.3 and Corollary 1.4 of {{mr:261565}} (<https://eudml.org/doc/161976>).
+Equivalently, every open cover of the space is refined by a locally finite clopen cover.  The equivalence between the two definitions is shown in Lemma 1.3 and Corollary 1.4 of {{zb:0182.25501}} (<https://eudml.org/doc/161976>).
```

Comment:
```suggestion
Equivalently, every open cover of the space is refined by a locally finite clopen cover.  The equivalence between the two definitions is shown in Lemma 1.3 and Corollary 1.4 of {{zb:0182.25501}}.
```
The eudml link is available from the zbmath page, so it can be removed from here.